### PR TITLE
Minor typo fixes

### PR DESCRIPTION
**Pull request by martinpeck**
A few typo fixes to an excellent post.

### CODE REVIEW

File: public/blog/jsbin-toxic-part-4.md [hunk 1]
```diff
@@ -86,15 +86,15 @@ Launching became the well know problem of chasing the end of the rainbow. Strivi
...
 
-Pro (eventually) went live on **23 July 2014**. And the first way of registrations was an amazing feeling. In fact, one user ([David Gauld](https://twitter.com/dcgauld), who actually worked for me doing the [Left Logic](http://leftlogic.com) redesign and built a fair amount of [confwall](https://confwall.com)) was the first to go pro catching the [commit landing in Github](https://github.com/jsbin/jsbin/commit/814251af40334990cf2490dc45d88ba246542f2f#diff-0aab8fc4f1799dc88c8d29729b719d87L118), even before I had announced it - a cool side effect of the open source.
+Pro (eventually) went live on **23 July 2014**. And the first wave of registrations was an amazing feeling. In fact, one user ([David Gauld](https://twitter.com/dcgauld), who actually worked for me doing the [Left Logic](http://leftlogic.com) redesign and built a fair amount of [confwall](https://confwall.com)) was the first to go pro catching the [commit landing in Github](https://github.com/jsbin/jsbin/commit/814251af40334990cf2490dc45d88ba246542f2f#diff-0aab8fc4f1799dc88c8d29729b719d87L118), even before I had announced it - a cool side effect of the open source.
```
Comment (martinpeck) [inline public/blog/jsbin-toxic-part-4.md]: You had "first way" and I think you meant "first wave"

File: public/blog/jsbin-toxic-part-4.md [hunk 2]
```diff
@@ -86,15 +86,15 @@ Launching became the well know problem of chasing the end of the rainbow. Strivi
...
 
-Users did come, but looking back, it really wasn't very much if you consider the costs of running JS Bin, let alone paying for development, and then look at what the monthly (or yearly subscriptions) go us. A total of 42 upgrades in the first week (26 were on the first day).
+Users did come, but looking back, it really wasn't very much if you consider the costs of running JS Bin, let alone paying for development, and then look at what the monthly (or yearly subscriptions) got us. A total of 42 upgrades in the first week (26 were on the first day).
```
Comment (martinpeck) [inline public/blog/jsbin-toxic-part-4.md]: you finish the sentence with "go us" and I think it should be "got us".

File: public/blog/jsbin-toxic-part-4.md [hunk 3]
```diff
@@ -86,15 +86,15 @@ Launching became the well know problem of chasing the end of the rainbow. Strivi
...
 
 ![JS Bin Stripe report, week 1](/images/jsbin-pro-week1.png)
 
 I had always tucked money aside for a rain day (actually rainy year), and yeah, I can look back and say "sure, I took a shot" - but it came at a pretty penny.
 
-The next 6 months I lost my love for the project. Very simply: any new development had to justify it's existence and it's demand on my time. I did continue development until the end of 2014, but by that point, all love had been sucked dry and I needed a break. JS Bin is solid enough that it can run without day to day attention and I could return to client work and *try* to recover my now sad looking business bank balance.
+The next 6 months I lost my love for the project. Very simply: any new development had to justify its existence and its demand on my time. I did continue development until the end of 2014, but by that point, all love had been sucked dry and I needed a break. JS Bin is solid enough that it can run without day to day attention and I could return to client work and *try* to recover my now sad looking business bank balance.
```
Comment (martinpeck) [inline public/blog/jsbin-toxic-part-4.md]: I've removed the apostrophes from two "it`s" as they're not needed here. Yes - I hate myself for pointing this out.

File: public/blog/jsbin-toxic-part-4.md [hunk 4]
```diff
@@ -116,4 +116,4 @@ The next time around, with products like [Confwall](https://confwall.com), I lau
 
 ---
 
-Part 5 sees our conclusion and tells of the police encounters I've had, along with the worst email I've ever received.
\ No newline at end of file
+Part 5 sees our conclusion and tells of the police encounters I've had, along with the worst email I've ever received.
```
Comment (martinpeck) [inline public/blog/jsbin-toxic-part-4.md]: Sorry - github picked this line up as altered, but I've not touched it. I have to assume it's a whitespace change. I should have cleaned it up before creating the PR. Soz.

File: public/blog/jsbin-toxic-part-4.md (hunk 3 above)
Comment (remy) [inline public/blog/jsbin-toxic-part-4.md]: <3

**Pull request merged**